server/src/api/community.rs

```diff
@@ -822,7 +823,10 @@ impl Perform for TransferCommunity {
 
     let mut admins = blocking(pool, move |conn| UserView::admins(conn)).await??;
 
-    let creator_index = admins.iter().position(|r| r.id == site_creator_id).unwrap();
+    let creator_index = admins
+      .iter()
+      .position(|r| r.id == site_creator_id)
+      .context("missing creator")?;
```

Comment:
You should probably still use the `location_info!()` macro, because if the error case happens, it will give no indication at all where in the code it was. That should be fine if its a unique string that we can grep for, but not if the same string is used multiple times.